pkg/repo/chartrepo_test.go

```diff
@@ -40,37 +40,22 @@ const (
 	testURL        = "http://example-charts.com"
 )
 
-func TestLoadChartRepository(t *testing.T) {
-	r, err := NewChartRepository(&Entry{
-		Name: testRepository,
-		URL:  testURL,
-	}, getter.All(&cli.EnvSettings{}))
-	if err != nil {
-		t.Errorf("Problem creating chart repository from %s: %v", testRepository, err)
-	}
+// loadFromDir a directory of charts archives (including sub-directories),
+// appending to the repositores ChartPath
+func loadFromDir(t *testing.T, r *ChartRepository, dir string) {
```

Comment:
I rewrote and simplified the original `Load()` function to this test helper, as it is still needed for `TestIndex`